common/changes/@rushstack/heft-config-file/benpapp-upgrade-ts-4.7_2022-08-02-20-58.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Upgrading TypeScript dependency to 4.7",
+      "type": "minor",
```

```suggestion
      "comment": "",
      "type": "none",
```

This doesn't matter to consumers.